client/ayon_blender/plugins/create/create_blendScene.py

```diff
@@ -32,3 +32,17 @@ def create(
                     instance_node.objects.link(data)
 
         return instance_node
+
+    def remove_instances(self, instances):
+
+        for instance in instances:
+            node = instance.transient_data["instance_node"]
+
+            if isinstance(node, bpy.types.Collection):
+                for children in node.children_recursive:
+                    if isinstance(children, bpy.types.Collection):
+                        bpy.data.collections.remove(children)
+                    else:
+                        bpy.data.objects.remove(children)
```

Comment:
Would just be this? (assuming the object is a collection)
```suggestion
            bpy.data.collections.remove(node)
```

Reply:
indeed yes